src/tsickle.ts

```diff
@@ -1312,7 +1312,7 @@ class Annotator extends ClosureRewriter {
     const name = getIdentifierText(iface.name);
     this.emit(`function ${name}() {}\n`);
 
-    this.emit(`\n\nfunction ${name}_tsickle_Closure_declarations() {\n`);
+    this.emit(`\n\nif (false) {\n`);
```

Comment:
Argh.  Given that it took me like a week of TAP train to get this one working, how about I submit this (to prevent regressions) and then do that next?

Reply:
Actually, the place I modified is just for 'interface' decls, where it's probably safe to emit without the 'if false' entirely.  I am gonna just do all of this in one CL, I take back my above comment.